### DISCUSSION TITLE
Maps Index on Website

### DISCUSSION THREAD

**Discussion by DanVanAtta** [triplea-game/triplea#10837] [General]
@DanVanAtta I mean if you want i can recreate this for the live maps server no problem.
The easiest approach would be to create a GitHub action on the website repo that runs every day or so. This way the code only relies on the HTTP API not on the underlying implementation.
Alternatively we could also implement this directly into the code whenever the database is updated, but I think even though it is more efficient it's the worse choice overall

_Originally posted by @RoiEXLab in https://github.com/triplea-game/triplea/issues/10781#issuecomment-1167012351_

**Comment by DanVanAtta** [comment]
@RoiEXLab , I think any of the options you mentioned would work. I really wonder if it is worth it:
1) initial effort
2) the complexity
3) the processing time
4) low return, almost no visits to the maps section

Traffic to the website is quite anemic, AFAIK there are close to zero visits to the maps section per month. I think maybe we might be better to focus on the in-game maps selection (tags etc). The website could use some sort of content that is somewhat dynamic, maybe like a blog post or news section. The maps section has issues.

**Comment by TheDog-GH** [comment]
FWIW & my 2p.
**Save the time and effort for other projects.**

As I understand it the https://triplea-game.org/maps-list/maps/ does not match the in game download list anyway.

If you want something for that maps page just list the 4-5 map games on https://triplea-game.org/ to showcase WW2, Napoleonic's & Ancients etc.

Im still hoping that 2.6 map downloads will have map tags (maps defined by era/theatre, star rating etc) to help players find maps more easily.

**Reply by DanVanAtta** [reply]
@TheDog-GH  - thanks for the input. Couple clarifications:
- The `maps-list/maps` page should match the in-game download list (at least, as of 2.5). For 2.6, the source of data changes and it will become out of sync unless we rebuild the sync job.
- 2.6 already has map tags added as a feature.

As far as showcasing some highlights, that could be a good idea. We could focus our content to be more in-depth reviews and perhaps walk-throughs of the game, content that would more likely to be read.

**Reply by TheDog-GH** [reply]
@DanVanAtta 
Is there a panel/url for entering and editing the map tag data?

If so, what is their format and is there a limit to how many characters of tags per map?

Im not sure when you plan to release 2.6, but it looks like its getting close, so from my point of view it would be good to enter some tags for testing, before public release.  There are about 172 maps, each map could have say 3 or more tags, so over 500 Tags to enter.

**Reply by DanVanAtta** [reply]
> Is there a panel/url for entering and editing the map tag data?

Late reply here - yes. IIRC it was part of the admin toolbox.. I think I'll need to double check where that is, I'll let you know when I run across it.
